### PR TITLE
srclib-python: install python2.7 in Docker

### PR DESCRIPTION
**Pull request by Unknwon**
The change is not ideal, someone please help make some comments or propose changes to srclib-python/Makefile.

### CODE REVIEW

File: services/worker/dockerfiles/Dockerfile.srclib-python [hunk 1]
```diff
@@ -20,6 +20,10 @@ RUN mkdir -p $SRCLIBPATH/sourcegraph.com/sourcegraph && mv /toolchain/t $SRCLIBP
 WORKDIR $SRCLIBPATH/sourcegraph.com/sourcegraph/srclib-python
 RUN make install
 
+RUN virtualenv -p python2.7 .env \
```
Comment (beyang) [inline services/worker/dockerfiles/Dockerfile.srclib-python]: How about doing it in the Makefile like this instead: https://github.com/sourcegraph/srclib-python/commit/ea16c866f16583a04821bd2355e0c194434cc29b

Comment (Unknwon) [inline services/worker/dockerfiles/Dockerfile.srclib-python]: The problem is we need to install once for Python3 and another one for Python2, if we set `.env` for them both time, only the last one is installed (Python3 in this case), vice versa.

That's why we need to change `.env` only after one version is installed.

**Comment by beyang** [comment]
@Unknwon see https://github.com/sourcegraph/sourcegraph/commit/d76af33730606b23e4fb20c051e12b1d20f88e2b. This is the only change necessary now that the srclib-python Makefile installs both versions of Python.

**Comment by beyang** [comment]
Lgtm

**Pull request merged**